### PR TITLE
Only refresh invalid lines again

### PR DESCRIPTION
**Pull request by jixiuf**

### CODE REVIEW

File: vterm-module.c [hunk 1]
```diff
@@ -138,125 +138,124 @@ static size_t get_col_offset(Term *term, int row, int end_col) {
...
-    }
-    delete_lines(env, liner, 1, false);
-    goto_line(env, liner);
-    refresh_row(term, env, r, width, false);
-  }
+  goto_line(env, line_start);
+  delete_lines(env, line_start, term->invalid_end - term->invalid_start, true);
+  refresh_lines(term, env, term->invalid_start, term->invalid_end, width);
+
+end:
```
Comment (akermu) [inline vterm-module.c]: Can you change this to a if statement?

**Comment by deb0ch** [comment]
What does this PR do exactly ?

**Comment by VanLaser** [comment]
Optimization, don't redraw everything, only the modified lines.

**Comment by deb0ch** [comment]
I can confirm that the performance boost is _visible_.

Input used to be very laggy before this PR was merged when the previous commands had a lot of output, now it is blazing fast ⚡️

I just tested again on the commits before and after, the diff is huge.

**Pull request merged**